static/app/components/replays/diff/picker/mutationOption.tsx

```diff
@@ -0,0 +1,63 @@
+import styled from '@emotion/styled';
+
+import {Flex} from 'sentry/components/container/flex';
+import Radio from 'sentry/components/radio';
+import {IconClock} from 'sentry/icons/iconClock';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import formatDuration from 'sentry/utils/duration/formatDuration';
+import {EventType, type RecordingFrame} from 'sentry/utils/replays/types';
+
+interface Props {
+  frame: RecordingFrame;
+  isChecked: boolean;
+  onChange: (e: React.FormEvent<HTMLInputElement>) => void;
+  radioName: string;
+  startTimestampMs: number;
+}
+
+export default function MutationOption({
+  frame,
+  startTimestampMs,
+  isChecked,
+  onChange,
+  radioName,
+}: Props) {
+  const name =
+    frame.type === EventType.FullSnapshot ? t('Full Snapshot') : t('Inc. Snapshot');
```

Comment:
I don't think these are great labels: `Inc.` is unclear what it means. Would our users know what `Full Snapshot` means either? Maybe we don't need `Snapshot`? just `Full` vs `Incremental` or `Full` vs `Mutation`?

Reply:
ya. i can trim them down... i added the mostly so i could better understand what is the data i'm look at. They might even be meaningless to someone debugging. I'm wide open on this and what we should show inside these buttons

Reply:
It's going in the right direction.

![SCR-20250107-khfd](https://github.com/user-attachments/assets/0b9064f5-2323-4426-b950-b4ae350f0253)

Reply:
are there docs about when rrweb will do a fullsnapshot vs incremental? that could help a lot

Reply:
Generally, when recording starts, it takes a snapshot of the DOM. Everything else are incremental updates to the snapshot.

(Exceptions being that you can tell it to take a full snapshot).

Reply:
cc @Jesse-Box